tests/kamereon/test_kamereon_vehicle_data.py

```diff
@@ -329,6 +328,36 @@ def test_location_v2() -> None:
     assert vehicle_data.gpsLongitude == 11.1234567
     assert vehicle_data.lastUpdateTime == "2020-02-18T16:58:38Z"
 
+def test_lock_status() -> None:
+    """Test lock-status for lock-status.json."""
+    response: models.KamereonVehicleDataResponse = fixtures.get_file_content_as_schema(
+        f"{fixtures.KAMEREON_FIXTURE_PATH}/vehicle_data/lock-status.json",
+        schemas.KamereonVehicleDataResponseSchema,
+    )
+    response.raise_for_error_code()
+    assert response.data is not None
+    assert response.data.raw_data["attributes"] == {
+        "lockStatus": "locked",
+        "doorStatusRearLeft": "closed",
+        "doorStatusRearRight": "closed",
+        "doorStatusDriver": "closed",
+        "doorStatusPassenger": "closed",
+        "hatchStatus": "closed",
+        "lastUpdateTime": "2022-02-02T13:51:13Z"
+    }
+
+    vehicle_data = cast(
+        models.KamereonVehicleLockStatusData,
+        response.get_attributes(schemas.KamereonVehicleLockStatusDataSchema),
+    )
+
+    assert vehicle_data.lockStatus == "locked"
+    assert vehicle_data.doorStatusRearLeft == "closed"
+    assert vehicle_data.doorStatusRearRight == "closed"
+    assert vehicle_data.doorStatusDriver == "closed"
+    assert vehicle_data.doorStatusPassenger == "closed"
+    assert vehicle_data.hatchStatus == "closed"
+    assert vehicle_data.lastUpdateTime == "2022-02-02T13:51:13Z"
```

Comment:
Missing carriage return
```suggestion
    assert vehicle_data.lastUpdateTime == "2022-02-02T13:51:13Z"

```